src/utils.js

```diff
@@ -284,6 +284,21 @@ export const convertToStringFromDate = (date) => {
   return moment(date).format(DATE_TIME_FORMAT);
 };
 
+export const convertToStringFromDateAndFormat = (date, format) => {
+  /**
+   * Convert local time to UTC string from react-datepicker in a format
+   * Note: react-datepicker has a bug where it only interacts with local time
+   * @param {Date} date - date in local time
+   * @param {string} format - format of the date
+   * @return {string} date converted in string in the desired format
+   */
+  if (!date) {
+    return '';
+  }
+
+  return moment(date).format(format);
```

Comment:
Are we supposed to use `moment` instead of [`FormattedDate`](https://formatjs.io/docs/react-intl/components/#formatteddate)? I think it's better to use the latter, but I'm not sure. Will moment be localized correctly?

Reply:
Yeah, I see we're using `FormattedDate` in other places. Please use that here too.

Reply:
Updated: https://github.com/openedx/frontend-app-course-authoring/pull/1138/commits/8d156e319adbb614e7b945a07887d0e978c6eb37